tests/results/scale/edge/edge-plus.md

```diff
@@ -148,21 +148,21 @@ The logs are attached only if there are errors.
 
 ### Event Batch Processing
 
-- Total: 104
-- Average Time: 117ms
+- Total: 43
+- Average Time: 159ms
 - Event Batch Processing distribution:
-	- 500ms: 104
-	- 1000ms: 104
-	- 5000ms: 104
-	- 10000ms: 104
-	- 30000ms: 104
-	- +Infms: 104
+	- 500ms: 41
+	- 1000ms: 43
+	- 5000ms: 43
+	- 10000ms: 43
+	- 30000ms: 43
+	- +Infms: 43
 
 ### Errors
 
-- NGF errors: 1
+- NGF errors: 0
```

Comment:
should we remove the log fles since there are no errors?

Reply:
Yeah probably